classes/operations/book_manager.rb

```diff
@@ -0,0 +1,105 @@
+require 'json'
+require_relative '../structure/book'
+require_relative '../../module/data_operations'
+
+class Bookmanager
```

Comment:
How about we abstract the `book creation process` (including the respective user input requests) to a separate module that we can call `book_creation` or something similar and leave out the `list_books` and `add_new_book` methods as the main operations of the `Book Manager`.

We could also move the book menu and its input options to a separate module we can call `book_submenu` or `book_menu`

This would help to assign single responsibilities to our files :100: